build.sbt

```diff
@@ -1139,10 +1139,32 @@ lazy val `sbt-plugin` = (project in file("dev") / "sbt-plugin")
     ),
     scriptedDependencies := {
       val () = scriptedDependencies.value
+
+      // javadsl projects
+      val () = (publishLocal in `api-javadsl`).value
+      val () = (publishLocal in `client-javadsl`).value
+      val () = (publishLocal in `service-registration-javadsl`).value
+      val () = (publishLocal in `service-registry-client-javadsl`).value
+
+      // core projects
+      val () = (publishLocal in api).value
+      val () = (publishLocal in spi).value
+      val () = (publishLocal in logback).value
+      val () = (publishLocal in client).value
+      val () = (publishLocal in server).value
+      val () = (publishLocal in jackson).value
+
+      // dev environment projects
+      val () = (publishLocal in `reloadable-server`).value
+      val () = (publishLocal in `sbt-build-tool-support`).value
       val () = publishLocal.value
+      val () = (publishLocal in `dev-mode-ssl-support`).value
       val () = (publishLocal in `service-locator`).value
-      val () = (publishLocal in LocalProject("sbt-scripted-tools")).value
+      val () = (publishLocal in `service-registry-client-core`).value
+
+      // sbt scripted projects
       val () = (publishLocal in `sbt-scripted-library`).value
+      val () = (publishLocal in LocalProject("sbt-scripted-tools")).value
```

Comment:
@dwijnand, to make sure I understand the point here...

when calling
```
val () = (publishLocal in someProject).value
```
we force the evalutaion and therefore we for it to be published locally, correct?

Why do we need more of it now? How was this working before? Transitively?

Reply:
It only worked if you ran `publishLocal`.

Reply:
Before this PR: if you change some code somewhere in Lagom and you want it to be picked up by `scripted` tests you have to manually invoke `publishLocal`.

That is, it wasn't working before.

Reply:
ok, got it